nox/virtualenv.py

```diff
@@ -29,6 +29,14 @@
     ["PIP_RESPECT_VIRTUALENV", "PIP_REQUIRE_VIRTUALENV", "__PYVENV_LAUNCHER__"]
 )
 _SYSTEM = platform.system()
+try:
+    # This attribute is set at runtime when running under a frozen
+    # pyinstaller binary
+    sys.frozen
+    _FROZEN = True
+except AttributeError:
+    # Catching an AttributeError is actually faster than (get|has)attr()
+    _FROZEN = False
```

Comment:
1.  What about instead using?
    ```python
    _FROZEN = getattr(sys, 'frozen', None) is not None
    ```
1.  ISTM that `_FROZEN` isn't specific enough. `_SYS_FROZEN` instead? Or `_SYS_HAS_FROZEN`?

Reply:
Catching the attribute exception is actually more performant than `hasattr`, but I'll change.